### PR TITLE
PEP 621: Clearly specify that PEP 621 is not meant  to be read from an sdist

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: pep-0621.rst [hunk 1]
```diff
@@ -49,6 +50,15 @@ This PEP is also not trying to change the underlying `core metadata`_
...
 may lead to changes or additions to what this PEP specifies.
 
+Finally, this PEP is meant for users to specify metadata for build
+back-ends or those doing analysis on a source checkout. Once a build
+back-end has been used to produce an artifact then the metadata
+contained in the artifact that the build back-end produced should be
+considered canonical and overriding what this PEP specifies. In the
+eyes of this PEP, an sdist is considered a build artifact, thus people
+should not read the metadata specified in this PEP as the canonical
+metadata in an sdist.
```
Comment (di) [inline pep-0621.rst:60]:
```suggestion
eyes of this PEP, a source distribution is considered a build
artifact, thus people should not read the metadata specified in this
PEP as the canonical metadata in a source distribution.
```

File: pep-0621.rst [hunk 2]
```diff
@@ -49,6 +50,15 @@ This PEP is also not trying to change the underlying `core metadata`_
 in any way. Such considerations should be done in a separate PEP which
 may lead to changes or additions to what this PEP specifies.
 
+Finally, this PEP is meant for users to specify metadata for build
+back-ends or those doing analysis on a source checkout. Once a build
+back-end has been used to produce an artifact then the metadata
```
Comment (di) [inline pep-0621.rst:55]:
```suggestion
back-end has produced an artifact, then the metadata
```

File: pep-0621.rst [hunk 3]
```diff
@@ -29,14 +29,15 @@ Motivation
 The key motivators of this PEP are:
 
 - Encourage users to specify core metadata statically for speed,
-  ease of specifying, and deterministic consuming
+  ease of specifying, deterministic consuming by build back-ends,
```
Comment (pganssle) [inline pep-0621.rst:32]:
```suggestion
  ease of specifying, deterministic consumption by build back-ends,
```

Might just be my regional dialect, but "consuming" as a noun like this doesn't sound right to me, I would use "consumption".

Comment (pfmoore) [reply pep-0621.rst:32]:
```suggestion
  ease of specification, deterministic consumption by build back-ends,
```

Agreed. I'd also use "specification" - for the same reason.

File: pep-0621.rst [hunk 4]
```diff
@@ -29,14 +29,15 @@ Motivation
 The key motivators of this PEP are:
 
 - Encourage users to specify core metadata statically for speed,
-  ease of specifying, and deterministic consuming
+  ease of specifying, deterministic consumption by build back-ends,
```
Comment (brettcannon) [inline pep-0621.rst:32]:
```suggestion
  ease of specification, deterministic consumption by build back-ends,
```

**Comment by brettcannon** [comment]
Thanks everyone!

**Pull request merged**